mod/admin.php

```diff
@@ -1422,6 +1434,9 @@ function admin_page_site(App $a)
 
 		'$check_new_version_url' => ['check_new_version_url', L10n::t("Check upstream version"), Config::get('system', 'check_new_version_url'), L10n::t("Enables checking for new Friendica versions at github. If there is a new version, you will be informed in the admin panel overview."), $check_git_version_choices],
 		'$suppress_tags'	=> ['suppress_tags', L10n::t("Suppress Tags"), Config::get('system','suppress_tags'), L10n::t("Suppress showing a list of hashtags at the end of the posting.")],
+		'$dbclean'		=> ['dbclean', L10n::t("Clean database"), Config::get('system','dbclean', false), L10n::t("Remove old remote items, orphaned database records and old content from some other helper tables.")],
```

Comment:
could you regenerate the `util/messages.po` please to get the new strings into the translation loop

Reply:
I know you told me - but I hadn't wrote it down. What is the command to do so?

Reply:
GOTO $friendica_dir
bin/run_xgettext.sh

This will by default regenerate the master messages.po file that you can include in your PR then.

Reply:
Done